image_writers_guide/sti_testing.adoc

```diff
@@ -10,64 +10,33 @@
 toc::[]
 
 == Overview
+As an STI builder image author, you can test your STI image locally and use the OpenShift build system for automated testing and continuous integration.
 
-This document covers how as a STI builder image author you can test your
-STI image locally and how you can use the OpenShift build system for automated
-testing and continuous integration.
+NOTE: See the link:sti.html[STI Requirements] topic to learn more about the STI architecture before reading further.
 
-Read more about the link:sti[STI architecture and requirements] before reading
-any further.
+As described in the link:sti.html[STI Requirements] topic, STI requires `assemble` and `run` scripts to be present in order to successfully execute the STI build. Providing the `save-artifacts` script reuses the build artifacts, and providing the `usage` script prints the usage to console when someone runs the Docker image outside of the STI.
```

Comment:
[Question]
"Prints the usage"? I'm not sure if I understand what we mean here.

[Suggestion]
s/STI requires/STI requires the

s/to console/to the console??